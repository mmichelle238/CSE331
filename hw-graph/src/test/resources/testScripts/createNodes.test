# Tests graph creation/viewing with a graph
# that has nodes that aren't connected to each other.

CreateGraph graph1
AddNode graph1 A
AddNode graph1 B
AddNode graph1 C
ListNodes graph1