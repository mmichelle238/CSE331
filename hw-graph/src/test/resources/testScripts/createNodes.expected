# Tests graph creation/viewing with a graph
# that has nodes that aren't connected to each other.

created graph graph1
added node A to graph1
added node B to graph1
added node C to graph1
graph1 contains: A B C